CMakeLists.txt

```diff
@@ -140,4 +152,4 @@ add_custom_target(check-clang  ${CI_DIR}/lint-clang.sh)
 add_custom_target(check-format DEPENDS check-clang check-python)
 
 # Check-all is just check, for now
-add_custom_target(check-all DEPENDS check)
+add_custom_target(tpp-check-all DEPENDS tpp-check)
```

Comment:
As observed below, better to just remove this target and make the test one `check-tpp`.